src/utils/institutions-helpers.ts

```diff
@@ -26,6 +26,23 @@ export const getOrganizationHierarchy = (unit?: Organization, result: Organizati
   return getOrganizationHierarchy(unit.partOf[0], [unit, ...result]);
 };
 
+export const findAncestor = (organization: Organization): Organization => {
+  if (organization.partOf && organization.partOf.length > 0) {
+    return findAncestor(organization.partOf[0]);
+  }
+  return organization;
```

Comment:
dette vil trolig gi samme også ved å gjenbruke den vi har fra før. Du kan vurdere selv om du syns det forenkler eller kompliserer. Your call, jeg bare kaster det ut her :) 
```suggestion
  return getOrganizationHierarchy(organization)[0];
```